api/tests/integration/infrastructure/repositories/target-profile-with-learning-content-repository_test.js

```diff
@@ -0,0 +1,156 @@
+      airtableBuilder.mockLists(airtableObjects);
+      const skill1_1_1_2 = domainBuilder.buildTargetedSkill({
+        id: 'recArea1_Competence1_Tube1_Skill2',
+        name: 'skill1_1_1_2_name',
+        tubeId: 'recArea1_Competence1_Tube1',
+      });
+      const skill1_2_1_1 = domainBuilder.buildTargetedSkill({
+        id: 'recArea1_Competence2_Tube1_Skill1',
+        name: 'skill1_2_1_1_name',
+        tubeId: 'recArea1_Competence2_Tube1',
+      });
+      const tube1_1_1 = domainBuilder.buildTargetedTube({
+        id: 'recArea1_Competence1_Tube1',
+        practicalTitle: 'tube1_1_1_practicalTitle',
+        competenceId: 'recArea1_Competence1',
+        skills: [skill1_1_1_2],
+      });
+      const tube1_2_1 = domainBuilder.buildTargetedTube({
+        id: 'recArea1_Competence2_Tube1',
+        practicalTitle: 'tube1_2_1_practicalTitle',
+        competenceId: 'recArea1_Competence2',
+        skills: [skill1_2_1_1],
+      });
+      const competence1_1 = domainBuilder.buildTargetedCompetence({
+        id: 'recArea1_Competence1',
+        name: 'competence1_1_name',
+        index: 'competence1_1_index',
+        areaId: 'recArea1',
+        tubes: [tube1_1_1],
+      });
+      const competence1_2 = domainBuilder.buildTargetedCompetence({
+        id: 'recArea1_Competence2',
+        name: 'competence1_2_name',
+        index: 'competence1_2_index',
+        areaId: 'recArea1',
+        tubes: [tube1_2_1],
+      });
+      const area1 = domainBuilder.buildTargetedArea({
+        id: 'recArea1',
+        title: 'area1_TitleFr',
+        competences: [competence1_1, competence1_2],
+      });
+      const expectedTargetProfile = databaseBuilder.factory.buildTargetProfile();
+      databaseBuilder.factory.buildTargetProfileSkill({ targetProfileId: expectedTargetProfile.id, skillId: 'recArea1_Competence1_Tube1_Skill2' });
+      databaseBuilder.factory.buildTargetProfileSkill({ targetProfileId: expectedTargetProfile.id, skillId: 'recArea1_Competence2_Tube1_Skill1' });
+      await databaseBuilder.commit();
+
+      // when
+      const targetProfile = await targetProfileWithLearningContentRepository.get(expectedTargetProfile.id);
+
+      // then
+      expect(targetProfile).to.be.instanceOf(TargetProfileWithLearningContent);
+      expect(targetProfile.id).to.equal(expectedTargetProfile.id);
+      expect(targetProfile.name).to.equal(expectedTargetProfile.name);
+      expect(targetProfile.skills).to.have.lengthOf(2);
+      expect(targetProfile.skills).to.deep.include.members([skill1_1_1_2, skill1_2_1_1]);
+      expect(targetProfile.tubes).to.have.lengthOf(2);
+      expect(targetProfile.tubes).to.deep.include.members([tube1_1_1, tube1_2_1]);
+      expect(targetProfile.competences).to.have.lengthOf(2);
+      expect(targetProfile.competences).to.deep.include.members([competence1_1, competence1_2]);
+      expect(targetProfile.areas).to.have.lengthOf(1);
+      expect(targetProfile.areas).to.deep.include.members([area1]);
```

Comment:
faire un `expect(targetProfile).to.deep.equal(expectedTargetProfile)` serait pas plus facile à lire & avec moins de risque d'oublier des trucs ?